autoload/go/term.vim

```diff
@@ -104,7 +104,7 @@ function! s:on_exit(job_id, exit_status, event) dict abort
   endif
   let job = s:jobs[a:job_id]
 
-  let l:listtype = "locationlist"
+  let l:listtype = go#list#Type("_term", "locationlist")
```

Comment:
The `_term` value isn't documented; is that intentional?

Reply:
Yes. From the most recent commit message:
> When displaying the results of a command that was run in a terminal,
pass "_term" to go#list#Type.  _term should not be considered one of the
supported values of g:go_list_type_commands.